recipes/dsnparse/0001-Include-License-in-source-distribution.patch

```diff
@@ -0,0 +1,20 @@
+From 7ab57006a38816c13a9ef37a5df7aad63b136459 Mon Sep 17 00:00:00 2001
```

Comment:
Delete this patch and manually package the license file:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually